lib/Models/CzmlCatalogItem.js

```diff
@@ -29,7 +29,7 @@ var readJson = require('../Core/readJson');
 var CzmlCatalogItem =  function(terria, url) {
      CatalogItem.call(this, terria);
 
-    this._czmlDataSource = undefined;
+    this.dataSource = undefined;
```

Comment:
This property should be named with an underscore, and then exposed as a public, non-underscored property with a getter only.  See `ImageryLayerCatalogItem.imageryLayer`.  Otherwise, `CatalogMember.serializeToJson` will try to write the data source to the JSON representation of the catalog item in some situations, which we don't want.